src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs

```diff
@@ -632,5 +632,22 @@ public void ObjectDump()
                 view.ObjectDump();
             }
         }
+
+        /// <summary>
+        /// Search through this View's hierarchy for a View with the given unique ID.
+        /// </summary>
+        /// <param name="id">The ID of the View to find.</param>
+        /// <remarks>Hidden-API</remarks>
+        /// <returns>A handle to the View if found, or an empty handle if not.</returns>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public View FindChildByID(uint id)
```

Comment:
I wonder if child includes grandchildren, if true how about use `Descendant` instead child
```suggestion
        public View FindDescendantByID(uint id)
```